scripts/jenkins-yoctobuild-build.sh

```diff
@@ -466,6 +477,10 @@ if grep mender_servers <<<"$JOB_BASE_NAME"; then
                 $WORKSPACE/integration/extra/release_tool.py --set-version-of $docker --version pr
                 ;;
 
+            deviceadm)
+                # Abandonded microservice, do nothing
+                :
```

Comment:
haha, nice! Never seen it used like this before.

Reply:
I saw it for the first time few lines above, on `mender-client-qemu*` case :wink: